app/invitations/controllers.py

```diff
@@ -123,6 +124,59 @@ def send_request(new_user, committee):
         db.session.rollback()
         return Response.RequestError
 
+##
+## @brief      Sends an invitation to a committee-head to close
+##             a charge.
+##
+## @param      committee       The committee to join.
+## @param      new_user        The user to be invited.
+##
+## @return     True if email sent, False if not.
+##
+def send_close_request(user, committee, chargeID):
+    
+    admins = db.session.query(Users).filter(Users.is_admin == True).all()
+
+    admin_emails = []
+    for user in admins:
+        admin_emails.append(user.id + "@rit.edu")
+
+    invite = and_(
+        Invitations.user_name == committee.head,
+        Invitations.committee_id == committee.id,
+        Invitations.isInvite == False
+    )
```

Comment:
In other methods in this file `invite` gets used to see if the invite already exists. Right now this isn't being used, so it should be removed. If we want to prevent admins from getting multiple emails to close the same committee, we should check if this invite matches any invites in the database.